src/node/quoteverification.h

```diff
@@ -97,22 +97,23 @@ namespace ccf
       {
         case FAIL_VERIFY_OE:
           return jsonrpc::error(
-            jsonrpc::ErrorCodes::INTERNAL_ERROR, "Quote could not be verified");
+            jsonrpc::StandardErrorCodes::INTERNAL_ERROR,
```

Comment:
Should we have QUOTE_NOT_VERIFIED here? We have quite detailed otherwise, eg. for the code verification, this seems needlessly vague?